### DISCUSSION TITLE
Error on (first time) compile

### DISCUSSION THREAD

**Discussion by Cyo90** [SKCraft/Launcher#479] [I need help!] [answered]
I've been trying to compile the launcher for 2 hours without success.

I have uploaded the full scan log of the build here: https://scans.gradle.com/s/2t5ftxekashg4
Any idea how to fix it?

Also, using Windows 11 and JAVA JDK 8 (added to path)
EDIT: tried on another computer with W10 and JDK 8, got the same error.

![image](https://user-images.githubusercontent.com/21068334/181661058-99b02813-8158-4459-a51e-ec5968d088af.png)

**Comment by hedgehog1029** [comment]
well, you win the "Most unique error of the month" award

do you have enough free disk space? seems something has gone wrong either with the building of a launcher JAR or with the downloading of a dependency... possibly try a `./gradlew clean`?

**Reply by Cyo90** [reply]
Thanks for the answer. I tried yes, and got the same error :( i have enough disk space too (+200 GB)
I also tried to compile it on a different computer with Windows 10 (using the .bat and Java JDK 8) and i'm still getting the same error. Soooo maybe it's my fault and i'm doing something terribly wrong or idk 😅

I've used gradle on both computer previously, but i don't think it has something to do with it :/

**Comment by Cyo90** [comment, marked as answer]
Soooo I found a fix. It seems the error ocurres if you used gradle before.

Just delete the folder .gradle located at C:\Users\your_user\ 
Then open again the build.bat file and it should work.

I can finally work on my launcher now, yay!
